Source/Diagnostics/ReducedDiags/FieldProbeParticleContainer.H

```diff
@@ -37,14 +37,14 @@ struct ParticleVal
  * nattribs tells the particle container to allot 7 SOA values.
  */
 class FieldProbeParticleContainer
-    : public amrex::ParticleContainer<0,0,ParticleVal::nattribs>
+    : public amrex::ParticleContainer<0,0,FieldProbePIdx::nattribs>
```

Comment:
Aesthetics/readability:
```suggestion
    : public amrex::ParticleContainer<0, 0, FieldProbePIdx::nattribs>
```